qemu/tests/cfg/blockdev_commit_stress.cfg

```diff
@@ -0,0 +1,29 @@
+- blockdev_commit_stress:
+    type = blockdev_commit_stress
+    virt_test_type = qemu
+    only Linux
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    snapshot_tags = sn1 sn2 sn3 sn4
+
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+
+    image_name_sn2 = sn2
+    image_format_sn2 = qcow2
+
+    image_name_sn3 = sn3
+    image_format_sn3 = qcow2
+
+    image_name_sn4 = sn4
+    image_format_sn4 = qcow2
+
+    device_tag = "image1"
+    rebase_mode = unsafe
+    mount_point = "/tmp"
+    qemu_force_use_drive_expression = no
+    Windows:
```

Comment:
remove Windows as this tc is only for Linux

Reply:
Have updated it, thanks for pointing it out.